include/rocksdb/options.h

```diff
@@ -374,6 +374,14 @@ struct DBOptions {
   // Default: true
   bool paranoid_checks = true;
 
+  // If true, check on-disk WALs against WAL information stored in MANIFEST
+  // during recovery.
```

Comment:
"check" is not very descriptive to me. How about something like "If true, fail recovery if there is an inconsistency between WAL information in MANIFEST and actual WALs on disk"?

What sort of corruption might this catch that's not otherwise caught? (Don't we always want to report corruption?)

Reply:
Also, do we quietly ignore extra/untracked WALs or fail in that case?

Reply:
missing or corrupted WAL will be reported

Reply:
It's still not clear to me how extra/untracked WALs will be handled. Can a DB switch between using track_and_verify_wals_in_manifest=true and false?

Reply:
Yes, switches between true and false can happen. The verification will only verify those tracked WALs.